packages/ember-metal/lib/property_get.js

```diff
@@ -89,6 +95,14 @@ export function _getPath(root, path) {
   return obj;
 }
 
+function isGettable(obj) {
+  if (obj == null) {
+    return false;
+  }
+
+  return !!ALLOWABLE_TYPES[typeof obj];
```

Comment:
I don't think the explicit coercion is needed here.  It is probably fine either way though...